lib/util.js

```diff
@@ -62,17 +62,26 @@ exports.deprecate = function(fn, msg) {
   }
 
   var warned = false;
+  const warnedLines = new Set();
+
   function deprecated() {
+    var location = (new Error()).stack.match(/.*?\n.*\n.*?\((.*?:\d+):/)[1];
```

Comment:
@silverwind You are correct, I am rebasing now. I ll fix it in that. I didn't know that we will get lines like
it.
    at /Users/silverwind/git/droppy/server/server.js:77:45

this in the trace. Probably, I ll split the stack, take the second line and then parse it.